### DISCUSSION TITLE
Version Numbers in Marketplace

### DISCUSSION THREAD

**Discussion by seversol** [microsoft/vscode-pylint#586] [Q&A]
Hello,

I noticed that the [version numbers](https://marketplace.visualstudio.com/items?itemName=ms-python.pylint&ssr=false#version-history) in the marketplace for the extension differ from the [release numbers](https://github.com/microsoft/vscode-pylint/releases) on github. 

1. Is there a way to know what release/tag/commit in the github repo aligns to the version of the extension in marketplace?
2. Why are there so many more versions published in the marketplace (1 github "release" this year vs. 4 marketplace "versions" since Oct 7)?

Thanks for the help and for a very useful plugin!
-Scott

**Comment by karthiknadig** [comment]
The market place only shows the last 5 versions, you can look at VS Code's version view for more details. We only tag and create GH releases for stable versions:
![image](https://github.com/user-attachments/assets/7cfb31dd-451e-46aa-b799-8e378d170b0d)

Versions in that view that don't have `(pre-release)` are stable.
